### PR TITLE
Inventory 006 complete mongo db seeder

### PR DESCRIPTION
**Pull request by martinsung00**
**Size:** Small
**Related Trello Cards:** [card one](https://trello.com/c/tFpLfre7)
**Change(s):**
- Update seed.sql script to create indexes
- Update schema.sql to be fully functional
- Update readme
- Move seed_data outwards into seeder folder
- Added new scripts to support mongo seeding & creation of database and table in Postgres

**Notes:**
- New script now imports csv data directly into MongoDB (seed records)
- New seeder script for Postgres now supports indexes
- Successfully seeded 10 million records to MongoDB

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,13 +11,14 @@
...
     "test": "mocha test/**/*.test.js --require ./test/dom.js --require esm --require ignore-styles --require @babel/register ./test --exit || true",
+    "create-postgres-database": "psql -U TO_DO -d postgres -a -f ./src/gateways/schema/schema.sql",
     "seed-postgres": "psql -U TO_DO -d inventory -a -f ./seeder/postgres_seeder/seed.sql",
+    "seed-mongo": "mongoimport --type csv -d inventory -c Products --headerline --drop ./seeder/seed_data/mockData.csv",
     "generate-data": "babel-node -x \".js\" ./seeder/postgres_seeder/data_generator/index.js",
     "react-dev": "webpack -d --watch",
     "build": "webpack --mode production"
   },
   "dependencies": {
-    "@babel/plugin-transform-runtime": "^7.12.1",
```
Comment (martinsung00) [inline package.json:20]: **Annotation:** Babel's transform runtime plugin should fall under development dependencies.

File: seeder/postgres_seeder/data_generator/plugin.js [hunk 2]
```diff
@@ -11,14 +11,16 @@ export default class CSVGenerator {
...
+      headers: ["product_id", "style_id", "size", "quantity"],
+    });
 
     // Let developers know this method is running
     console.log(status.in_progress);
 
     writer.pipe(
       fs.createWriteStream(
-        `./seeder/postgres_seeder/data_generator/seed_data/mockData.csv`,
+        "./seeder/seed_data/mockData.csv",
```
Comment (martinsung00) [inline seeder/postgres_seeder/data_generator/plugin.js:23]: **Annotation:** moved seed_data folder outside.

File: test/src/seeder/plugin.test.js [hunk 3]
```diff
@@ -1,7 +1,6 @@
 import sinon from "sinon";
 import { expect } from "chai";
 import CSVGenerator from "../../../seeder/postgres_seeder/data_generator/plugin";
-import csvWriter from "csv-write-stream";
```
Comment (martinsung00) [inline test/src/seeder/plugin.test.js:4]: **Annotation:** Remove unused code.

**Review by TandyN — APPROVED** [review]
I see no major changes, mostly refactoring/cleaning.

**Pull request merged**